wallet-new/integration/Error.hs

```diff
@@ -21,51 +22,64 @@ import           Cardano.Wallet.Client (ClientError)
 data WalletTestError
     = HttpClientError ClientError
 
-    | InvalidProbabilityDistr
-
     | WalletBalanceNotZero Wallet
     | WalletPassMissing Wallet
-    | LocalWalletDiffers Wallet
-    | LocalWalletsDiffers [Wallet]
+    | LocalWalletDiffers Wallet Wallet
+    | LocalWalletsDiffers [Wallet] [Wallet]
 
     | AccountBalanceNotZero Account
-    | LocalAccountDiffers Account
-    | LocalAccountsDiffers [Account]
+    | LocalAccountDiffers Account Account
+    | LocalAccountsDiffers [Account] [Account]
 
     | AddressBalanceNotZero WalletAddress
     | LocalAddressesDiffer WalletAddress [WalletAddress]
-    | LocalAddressDiffer Address
+    | LocalAddressDiffer Address Address
 
     | InvalidTransactionState Transaction
     | InvalidTransactionFee EstimatedFees
-    | LocalTransactionsDiffer [Transaction]
+    | LocalTransactionsDiffer [Transaction] [Transaction]
```

Comment:
The error constructors now contain an `expected` and `actual` value, where a diff can be present.